contrib/benitlux/src/client/features/debug.nit

```diff
@@ -24,7 +24,7 @@ redef class UserWindow
 	private var layout_debug = new VerticalLayout(parent=layout)
 
 	private var lbl_debug_title = new Label(parent=layout_debug,
-		text="Debug options".t)
+		text="Debug options".t, size=1.5)
```

Comment:
No, it's OK. The init was left empty except for this line, I moved it up so it is cleaner. Moving a single line for aesthetic purposes as a direct side effect of this PR is not worth a dedicated PR.